apps/shared/avifjpeg.h

```diff
@@ -11,7 +11,8 @@ extern "C" {
 #endif
 
 // Decodes the jpeg file at path 'inputFilename' into 'avif'.
-// At most imageSizeLimit pixels will be read or an error returned.
+// At most sizeLimit pixels will be read or an error returned. At most sizeLimit
+// bytes of Exif or XMP metadata will be read or an error returned.
```

Comment:
[This is just a comment.] Thank you for documenting the two purposes of `sizeLimit`. Using `sizeLimit` for both the (uncompressed) image area and the Exif/XMP metadata size still has a conceptual difficulty: these two quantities have different units. The image area is in pixels, and the Exif/XMP metadata size is in bytes. In addition to the comment, we can clarify this by noting that `sizeLimit` ultimately limits the amount of memory used by the function.